flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/ExprCodeGenerator.scala

```diff
@@ -722,9 +722,9 @@ class ExprCodeGenerator(ctx: CodeGeneratorContext, nullableInput: Boolean)
       case ARRAY_VALUE_CONSTRUCTOR =>
         generateArray(ctx, resultType, operands)
 
-      // maps and multisets
-      case MAP_VALUE_CONSTRUCTOR | MULTISET_VALUE =>
-        generateMapOrMultiset(ctx, resultType, operands)
+      // maps
+      case MAP_VALUE_CONSTRUCTOR =>
```

Comment:
@snuyanzin Looking forward to a PR for this commit. Maybe you can also address the comment about the value constructor in this PR (using HashMap for deduplication of key).

Reply:
hi @twalthr , thanks for highlighting this
yes I will try to handle this